test/utils/test-constants.js

```diff
@@ -3,8 +3,8 @@
 var path = require('path');
 
 // Input/output relative paths
-var inputRelative = './fixtures/';
-var outputRelative = './out-fixtures/';
+var inputRelative = './fixtures';
```

Comment:
I just followed the same pattern as the corresponding bump in vinyl-fs,
https://github.com/gulpjs/vinyl-fs/commit/f775cff17f2f6564d718aa2d3de4d9b098a6e421#diff-6f48deb9b860a608625ec0870ed5c3aaR6
This made all but one of the tests pass, so all seemed well.